### PR TITLE
Add Helix rev3 default keymaps

### PR DESCRIPTION
**Pull request by MakotoKurauchi**
Add the default keymap for Helix rev3 following qmk/qmk_firmware#10297.

Thanks!

### CODE REVIEW

File: public/keymaps/h/helix_rev3_4rows_default.json [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      "KC_F7",
+      "KC_F8",
+      "KC_F9",
+      "KC_F10",
+      "KC_F11",
+      "KC_F12",
+      "KC_TRNS",
+      "RESET",
+      "KC_TRNS",
+      "KC_TRNS",
```
Comment (noroadsleft) [inline public/keymaps/h/helix_rev3_4rows_default.json:181]:
```suggestion
      "EEP_RST",
```

File: public/keymaps/h/helix_rev3_5rows_default.json [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+      "KC_F7",
+      "KC_F8",
+      "KC_F9",
+      "KC_F10",
+      "KC_F11",
+      "KC_F12",
+      "KC_TRNS",
+      "RESET",
+      "KC_TRNS",
+      "KC_TRNS",
```
Comment (noroadsleft) [inline public/keymaps/h/helix_rev3_5rows_default.json:223]:
```suggestion
      "EEP_RST",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Just these two things; looks good otherwise!

File: public/keymaps/h/helix_rev3_5rows_default.json (hunk 2 above)
Comment (MakotoKurauchi) [reply public/keymaps/h/helix_rev3_5rows_default.json]: thanks

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**